.github/workflows/coverage.yml

```diff
@@ -0,0 +1,51 @@
+name: coverage
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 06 * * 1" # 6AM weekly on monday
```

Comment:
we want to run code coverage for every pr & master, just like unit tests

Reply:
This needs to use self-host.  Running the code coverage for more than 1 hour.

Reply:
ok. I can setup another self-host server

Reply:
also we don't need to wait for coverage result before merge PR